EIPS/eip-7892.md

```diff
@@ -59,37 +71,91 @@ To facilitate these changes on the execution layer, the `blobSchedule` object sp
     "max": 9,
     "baseFeeUpdateFraction": 5007716
   },
-  "1740693335": {
+  "12000000": {
     "target": 24,
     "max": 48,
     "baseFeeUpdateFraction": 5007716
   },
-  "1743285335": {
+  "12300000": {
     "target": 36,
     "max": 56,
     "baseFeeUpdateFraction": 5007716
   }
 }
 ```
 
-On the consensus layer, a new parameter is added to the configuration:
+### Consensus layer configuration
 
-```
-BLOB_SCHEDULE:
-  - EPOCH: 348618
+A new `BLOB_PARAMETER_SCHEDULE` field is added to consensus layer configuration, containing a sequence of entries representing blob parameter changes after `ELECTRA_FORK_EPOCH`.
+
+Entry rules:
+
+- One entry per fork that changes blob parameters (regular or BPO forks).
+- `EPOCH` and `MAX_BLOBS_PER_BLOCK` required for all entries.
+- `FORK_VERSION` required only for BPO forks (regular forks specify this in source).
+
+```yaml
+BLOB_PARAMETER_SCHEDULE:
```

Comment:
this name is still different... last consensus-spec entry was `BLOB_SCHEDULE`

Reply:
Restored in https://github.com/ethereum/EIPs/pull/9840.